pkg/sync/v14/data/ClusterRole.rbac.authorization.k8s.io/system-metrics-server.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:metrics-server
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - nodes
+  - nodes/stats
+  - namespaces
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - extensions
+  resources:
+  - deployments
```

Comment:
this seems odd - do we know why it's here?

Reply:
you are right this is not heeded, I will remove it. 
It got here from OpenShift old fork manifest: https://github.com/openshift/kubernetes-metrics-server/blob/master/deploy/resource-reader.yaml#L16
It was removed later in the upstream: https://github.com/kubernetes-sigs/metrics-server/pull/195